crates/swc_ecma_transforms_react/src/jsx/mod.rs

```diff
@@ -396,10 +401,33 @@ where
                     }
                 }
 
+                let args = vec![fragment.as_arg(), props_obj.as_arg()];
+
+                let args = if self.development {
+                    args.into_iter()
+                        .chain(
```

Comment:
I think `.chain(once(undefined(DUMMY_SP).as_arg())).chain(once(Lit::Bool(Bool {
                                    span: DUMMY_SP,
                                    value: use_jsxs,
                                })
                                .as_arg()))` would be better.

Reply:
OK 👍  Thank you for helping! I fixed in 5be5cf0